app/src/reducer.js

```diff
@@ -0,0 +1,18 @@
+import {intiateState, addChannel, removeChannel, addMessage} from './core.js';
+
```

Comment:
I'd like to merge core functions with reducer, as core functions should only be accessible through actions and it doesn't make sense to test them separately.
Wdyt?